packages/blade/src/components/Toast/ToastContainer.web.tsx

```diff
@@ -324,10 +332,17 @@ type ToastContainerProps = {
    * @default isMobile ? 16px : 24px
    */
   offsetBottom?: number;
+  /**
+   * Custom z-index value for the toast container.
+   * This is useful when you want to position toasts above other elements like modals.
+   *
+   * @default 2000
```

Comment:
default zIndex should be more than Modal. 

In the TOAST_Z_INDEX also add 1 liner 

```
// Because more than modal
const TOAST_Z_INDEX = 2001;
```